app/views/obligations.scala.html

```diff
@@ -119,6 +119,29 @@ <h2 id="declarations-heading" class="heading">@messages("obligations.declaration
 <!--
     Income source EOPS subsection
 -->
+@for(businessIncomeSource <- incomeSources.businessIncomeSources if !incomeSources.hasBusinessReportErrors){
```

Comment:
this will cause EOPS for business income to not be displayed if there any errors - too broad a check